upup/models/cloudup/resources/addons/storage-aws.addons.k8s.io/v1.15.0.yaml.template

```diff
@@ -29,7 +29,7 @@ kind: StorageClass
 metadata:
   name: kops-ssd-1-17
   annotations:
-    storageclass.beta.kubernetes.io/is-default-class: "true"
+    storageclass.beta.kubernetes.io/is-default-class: "{{ if .CloudConfig.AWSEBSCSIDriver.Enabled }}{{ else }}false{{ end }}"
```

Comment:
I have a patch in the works that will require some coordination here, both with the annotation name and with surrounding the creation of the _StorageClass_ objects in a conditional block.

Reply:
Yeah I really dislike the annotation path here for controlling which is the default. It made rolling out some best practices for new clusters + not breaking old oddly difficult back when I did this.

Reply:
See #10733 for my patch on the subject. We'll have to resolve a bunch of conflicts, depending on which one goes in first.

Reply:
We'll merge yours first